poi/src/main/java/org/apache/poi/util/LocaleID.java

```diff
@@ -374,7 +374,7 @@ public enum LocaleID {
     MN_MONG_MN(0x0C50, "mn_mong_mn", "mn-Mong-MN", "Mongolian (Traditional Mongolian, Mongolia)", 0, -1),
     DZ_BT(0x0C51, "dz_bt", "dz-BT", "Dzongkha (Bhutan)", 0, SUNDAY),
     TMZ_MA(0x0C5F, "tmz_ma", "tmz-MA", "tmz (Morocco)", 32759, -1),
-    QUZ_PE(0x0C6b, "quz_pe", "quz-PE", "Quechua (Peru)", 1252, -1),
+    QUZ_PE(0x0C6B, "quz_pe", "quz-PE", "Quechua (Peru)", 1252, -1),
```

Comment:
This isn't a spelling fix, it's left beneath all the spelling fixes. All but these 5 entries use uppercase hex...